simpleone/src/simpletemplate/Dashboard.js

```diff
@@ -174,24 +174,27 @@ export default function Dashboard() {
       <main className={classes.content}>
         <div className={classes.appBarSpacer} />
         <Container maxWidth="lg" className={classes.container}>
+          <Box pt= {1}>
+            <Grid container direction="row" justify="flex-end" xs={12} md={12} lg={12}>
+              <TextField id="search-bar-temp" label="Search" variant="outlined" />
+            </Grid>
+          </Box>
           <Box pt={4}>
             <Grid container spacing={3}>
-              <Grid item item xs={12} md={6} lg={8}>
+              <Grid item xs={12} sm={12} md={4} lg={3}>
                 <Paper className={fixedHeightPaper}>
                   <Box>
                     <Typography component="h1" variant="h3" color="secondary" noWrap className={classes.title}>
-                      Dans ta face, Gih.
```

Comment:
I think this removal will break core functionality. Can't pass the merge request until fixed.